R/example.R

```diff
@@ -435,42 +440,53 @@ example_jira_issue_components <- function(folder_path="/tmp",folder_name) {
 #' @return the JSON path of the newly created issue issue tracker
 #' @export
 #' @keywords internal
-example_jira_two_issues <- function(folder_path="/tmp",folder_name) {
+example_jira_two_issues <- function(folder_path, folder_name) {
```

Comment:
Same problem. Make sure the default /tmp is not removed thereafter. I am not adding a comment for each instance so you will need to double check.